src/main/java/gregtech/common/blocks/BlockPanelling.java

```diff
@@ -0,0 +1,50 @@
+package gregtech.common.blocks;
+
+import gregtech.api.block.VariantBlock;
+
+import net.minecraft.block.SoundType;
+import net.minecraft.util.IStringSerializable;
+
+public class BlockPanelling extends VariantBlock<BlockPanelling.PanellingType> {
```

Comment:
These new block classes need to be actually registered in CommonProxy, both the blocks and the item blocks.

Reply:
Yep, just forgot to actually take the CommonProxy changes into the branch from `nuclear-fission`.